bundles/GoogleMarketingBundle/public/js/startup.js

```diff
@@ -0,0 +1,34 @@
+pimcore.registerNS('pimcore.bundle.google_marketing.startup');
+
+pimcore.bundle.google_marketing.startup = Class.create({
+    initialize: function () {
+        document.addEventListener(pimcore.events.preMenuBuild, this.preMenuBuild.bind(this));
+    },
+
+    preMenuBuild: function (event) {
+        const menu = event.detail.menu;
+        const user = pimcore.globalmanager.get('user');
+        const perspectiveCfg = pimcore.globalmanager.get("perspective");
+
+        if (menu.marketing && perspectiveCfg.inToolbar("settings.marketingReports")
+            && user.isAllowed("reports") && user.isAllowed("system_settings")) {
+            menu.marketing.items.push({
+                text: t("marketing_settings"),
+                iconCls: "pimcore_nav_icon_marketing_settings",
+                itemId: 'pimcore_menu_marketing_settings',
+                handler: this.reportSettings,
+                priority: 30
+            });
+        }
+    },
+    reportSettings: function () {
+        try {
+            pimcore.globalmanager.get("reports_settings").activate();
+        }
+        catch (e) {
+            pimcore.globalmanager.add("reports_settings", new pimcore.report.settings());
```

Comment:
what happens when the custom reports bundle is not installed nor enabled? this won't work right?
It should still show all the other settings that are available.

the class pimcore.report.settings() also does not exist anymore with the bundle refacotoring of the custom reports